iterators.tex

```diff
@@ -6349,8 +6356,8 @@
 \oldtxt{\tcode{begin} and \tcode{end} are required to be amortized constant time.}
 
 \pnum
-\newtxt{Given an object \tcode{t} of type \tcode{T}, \oldtxt{the type \tcode{T} models}
-\tcode{Range\newtxt{<T>()}} \newtxt{is satisfied} if and only if}
+\newtxt{Given a value \tcode{t} of type \tcode{T}, \tcode{Range<T>()} is satisfied
+if and only if}
```

Comment:
I honestly don't know. I stole the wording from the specification of `swap`.

\e

On Wed, Aug 5, 2015, 8:01 AM Casey Carter <notifications@github.com> wrote:

> In iterators.tex
> <https://github.com/ericniebler/stl2/pull/81#discussion_r36311677>:
>
> > @@ -6349,8 +6356,8 @@
> >  \oldtxt{\tcode{begin} and \tcode{end} are required to be amortized constant time.}
> >
> >  \pnum
> > -\newtxt{Given an object \tcode{t} of type \tcode{T}, \oldtxt{the type \tcode{T} models}
> > -\tcode{Range\newtxt{<T>()}} \newtxt{is satisfied} if and only if}
> > +\newtxt{Given a value \tcode{t} of type \tcode{T}, \tcode{Range<T>()} is satisfied
> > +if and only if}
>
> Why do we care that overload resolution selects "a unary non-member
> function"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ericniebler/stl2/pull/81/files#r36311677>.
>